lib/beaker/tasks/quick_start.rb

```diff
@@ -0,0 +1,49 @@
+require 'beaker-hostgenerator'
+
+namespace :beaker_quickstart do
```

Comment:
nvm this comment. In light of the rake task sharing discussion below, I was mistaken about this being redundant.